### PR TITLE
Fix certs even if no workers are deployed

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

**Comment by russellb** [comment]
Please hold on this. I need to understand why this isn’t working automatically for masters first. 

Related to #260

**Comment by russellb** [comment]
I looked into it and now understand why this is required.

The docs were just updated last week to clarify that only the node client cert will be auto approved on an ongoing basis, and that some automation will always be required for the server certs.

https://bugzilla.redhat.com/show_bug.cgi?id=1720178

https://github.com/openshift/openshift-docs/commit/6912507d69af22bfb8bd49ac920ec84221b97afb

https://github.com/openshift/openshift-docs/pull/16060

File: 11_register_hosts.sh [hunk 1]
```diff
@@ -78,13 +78,13 @@ list_workers | make_bm_workers | tee $SCRIPTDIR/ocp/worker_crs.yaml
...
 
-# Check if file exists
-[ -s "$SCRIPTDIR/ocp/worker_crs.yaml" ] || exit 0
-
-oc --config ocp/auth/kubeconfig apply -f $SCRIPTDIR/ocp/worker_crs.yaml --namespace=openshift-machine-api
-
 # Run the fix_certs.sh script periodically as a workaround for
 # https://github.com/openshift-metalkube/dev-scripts/issues/260 This is only
 # required to approve certs for workers, as the master certs are approved
 # automatically during the bootstrap phase.
```
Comment (zaneb) [inline 11_register_hosts.sh]: Shouldn't we remove this last sentence from the comment again as well?

Comment (russellb) [reply 11_register_hosts.sh]: yes

**Pull request merged**